wpeverest/user-registration
# UR-631 Profile Picture Refactor

### All Submissions:






### Changes proposed in this Pull Request:

Previously, We were uploading profile picture first then submitting the form which causing some security issues as well as privacy issues, This PR is for Refactor Profile Picture according to new approach. 

### How to test the changes in this Pull Request:

1. Goto Edit Profile and upload profile picture in both case ajax submission and post submission. 
2. Check Security Issues.

### Types of changes:




### Other information:




### Changelog entry

> Refactor - Profile Picture now upload only after user registered.
